<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileVideoSourceOptionsWidget</class>
 <widget class="QWidget" name="FileVideoSourceOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="filePathLabel">
         <property name="text">
          <string>File path:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="filePathLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="openFileButton">
       <property name="text">
        <string>&amp;Open file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fileInfoButton">
       <property name="text">
        <string>&amp;Info</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
